### ISSUE TITLE
Horizontal line missing under socials

### ISSUE THREAD

**Issue by bosschaert** [hlxsites/zeiss#184]
current:
https://main--zeiss--hlxsites.hlx.page/de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/2023/neue-multifunktionsfabrik-in-wetzlar
<img width="1338" alt="Screenshot 2023-04-26 at 13 08 16" src="https://user-images.githubusercontent.com/226514/234570237-7420ef49-3af1-4467-91a1-2fda75e0c3d2.png">

expected:
https://www.zeiss.de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/2023/neue-multifunktionsfabrik-in-wetzlar.html
<img width="1358" alt="Screenshot 2023-04-26 at 13 08 23" src="https://user-images.githubusercontent.com/226514/234570171-d321e78c-dfe5-4fa6-9064-c1023f1b67e9.png">

**Comment by sdmcraft** [comment]
I think this is present on some pages and absent on others. Probably it can be based on an author provided setting.
See  item 2 of #126

**Comment by bosschaert** [comment]
Yes correct the horizontal line is not on https://www.zeiss.de/semiconductor-manufacturing-technology/news-und-events/smt-pressemeldung/ausgezeichnete-ausbildung-bei-zeiss.html
